tutorials/7-cosmjs/5-create-custom.md

```diff
@@ -31,120 +31,38 @@ This section assumes that you have a working Cosmos blockchain with its own modu
+You can choose which library you use to compile your Protobuf objects into TypeScript or JavaScript. Reproducing what [cosmjs-types](https://github.com/confio/cosmjs-types/blob/main/scripts/codegen.js) or [Stargate](https://github.com/cosmos/cosmjs/blob/main/packages/stargate/CUSTOM_PROTOBUF_CODECS.md) do is a good choice.
 
 ### Preparation
 
 This exercise assumes that:
 
-1. Your Protobuf definition files are in `./proto/myChain`.
-2. You want to compile them into TypeScript in `./client/src/types/generated`.
+1. The library you're creating is in 'myLib'.
+2. Your Protobuf definition files are in `./proto/`.
+3. You want to transpile them into TypeScript in `./src/codegen/`.
 
-Install `protoc` on your computer and its Typescript plugin in your project, possibly with the help of a Dockerfile:
-
-<CodeGroup>
-
-<CodeGroupItem title="Local">
+Install `telescope` on your computer globally:
 
 ```sh
-$ mkdir -p /usr/lib/protoc
-$ cd /usr/lib/protoc
-$ curl -L https://github.com/protocolbuffers/protobuf/releases/download/v21.7/protoc-21.7-linux-x86_64.zip -o protoc.zip
-$ unzip -o protoc.zip
-$ rm protoc.zip
-$ ln -s /usr/lib/protoc/bin/protoc /usr/local/bin/protoc
-$ npm install ts-proto@1.121.6 --save-dev
+npm install -g @cosmology/telescope
 ```
 
-Adjust to your preferred version, operating system, and CPU platform. For instance, on an Apple M1 you would use `protoc-21.7-osx-aarch_64.zip`.
-
-</CodeGroupItem>
-
-<CodeGroupItem title="Dockerfile">
-
-```Dockerfile
-FROM --platform=linux node:lts-slim as base
-ARG BUILDARCH
-
-ENV PROTOC_VERSION=21.7
-ENV TS_PROTO_VERSION=1.121.6
-
-FROM base AS platform-amd64
-ENV PROTOC_PLATFORM=x86_64
-
-FROM base AS platform-arm64
-ENV PROTOC_PLATFORM=aarch_64
-
-FROM platform-${BUILDARCH}
-
-RUN apt-get update
-RUN apt-get install curl unzip --yes
-
-# Install ProtoC
-RUN mkdir -p /usr/lib/protoc
-WORKDIR /usr/lib/protoc
-RUN curl -L https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-linux-${PROTOC_PLATFORM}.zip -o protoc.zip
-RUN unzip -o protoc.zip
-RUN rm protoc.zip
-RUN ln -s /usr/lib/protoc/bin/protoc /usr/local/bin/protoc
-
-# Install ts-proto
-RUN npm install --global ts-proto@${TS_PROTO_VERSION} --save-exact
-
-WORKDIR /
-
-ENTRYPOINT [ "protoc" ]
-```
-
-Then build the image:
-
-```sh
-$ docker build . -t ts-protoc
-```
-
-</CodeGroupItem>
-
-</CodeGroup>
-
 ---
 
-You can confirm the version you received. The executable is located in `./node_modules/protoc/protoc/bin/protoc`:
-
-<CodeGroup>
-
-<CodeGroupItem title="Local">
-
-```sh
-$ protoc --version
-```
-
-</CodeGroupItem>
-
-<CodeGroupItem title="Docker">
+You can confirm the version you received by running:
 
 ```sh
-$ docker run --rm -it \
-    ts-protoc --version
+$ telescope -v
```

Comment:
If `--version` exists, please use the long form so that it is more easily understandable by the person skimming the content. Sometimes `-v` means _verbose_.